plugins/loading/test/loading-asNumber.test.js

```diff
@@ -0,0 +1,327 @@
+const delay = ms => new Promise(r => setTimeout(r, ms))
```

Comment:
There are a lot more tests now.. I just duplicated them and tested for boolean in one file, and number in this one.